### ISSUE TITLE
Getting ellipsoid rotation in space 

### ISSUE THREAD

**Issue by YuriiHordiienko1987** [BenjaminNavarro/ellipsoid-fit#2]
Estimating the rotation (around X,Y,Z and is pepresented by 3 angles) of ellipsoid by using eigenvectors.
Due to the ellipsoid-fit library there is a challenge to assosiate each angle with corresponding radii in true order.
Three angles could be given in RAD or DEG.

**Comment by BenjaminNavarro** [comment]
I'm not sure what you are asking for. Could you elaborate a bit?

Do you want the library to also return a rotation matrix? If so, considering the radii are in no particular order, do you think it is still useful?

**Comment by YuriiHordiienko1987** [comment]
Of course, the true order is affected, then maybe good idea to have a deal with a current order and then define rotation according to additional criteria. How will the rotation matrix look like related to the current order? According to the necessity to calculate dip angle in my particular case or full 9 parameters of ellipsoid in general case the presence of rotation matrix will be useful anyway. Thanks for response.

**Comment by BenjaminNavarro** [comment]
I think the rotation matrix can be constructed from the eigen vectors. The column order should match the one of the eigen values and thus the radii.

Then you can extract the angle you need from this rotation matrix or convert it to a different orientation representation.

I'll try to update the library soon
